raiden_contracts/contracts/services/UserDeposit.sol

```diff
@@ -0,0 +1,142 @@
+pragma solidity ^0.5.2;
+
+import "raiden/Token.sol";
+import "raiden/Utils.sol";
+
+contract UserDepositContract is Utils {
+    string constant public contract_version = "0.1.0";
```

Comment:
Note that we currently go with the same version for all contracts and we also need to write a test for this - e.g. https://github.com/raiden-network/raiden-contracts/blob/d08a3abaffdd9ebd4b6cd0a4209d34a1a1328715/raiden_contracts/tests/test_token_network_registry.py#L17-L20 (can be done in another PR, an issue can be opened).
Also, the overall contracts version is written in the precompiled + deployment data only once per all contracts.

Having multiple contract version within the same package can complicate things -> we went with redeployment of all contracts when changing the overall `CONTRACTS_VERSION`. You may want individual versions at some point, but I think we should be consistent until the decision to change this is reached.